src/components/page-environment/style.css

```diff
@@ -7,15 +7,19 @@
   width: 100%;
 }
 
+.env__link {
+  color: inherit;
+}
+
```

Comment:
can be removed if `Typography` is put within `Link` component
```suggestion
```

Reply:
fixed